templates/base.html

```diff
@@ -1,126 +1,162 @@
+      <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://assets.ubuntu.com/v1/09460d9a-apple-touch-icon-72x72-precomposed.png" />
+      <link rel="apple-touch-icon-precomposed" href="https://assets.ubuntu.com/v1/cc66da65-apple-touch-icon-precomposed.png" />
 
-        <link rel="stylesheet" href="/static/css/main.css">
-    </head>
+      <link rel="stylesheet" href="/static/css/main.css">
+      <script src="/static/js/highlight.pack.js"></script>
+      <script>
+        // Set up highlightjs
+        hljs.initHighlightingOnLoad();
 
-    <body class="docs">
-        <!-- Google Tag Manager (noscript) -->
-        <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K92JCQ"
-        height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-        <!-- End Google Tag Manager (noscript) -->
+        // Run when DOM is ready...
+        document.addEventListener("DOMContentLoaded", function(event) {
+          // Toggle mobile sidebar nav
+          var toggle = document.querySelector('.p-sidebar__toggle');
+          var sidebarContent = document.querySelector('.p-sidebar__content');
 
-        <header class="banner global" role="banner">
-          <nav role="navigation" class="nav-primary" id="nav-primary">
-              <span class="accessibility-aid">
-                  <a href="#main-content">Jump to main content</a>
-              </span>
-              <div class="logo">
-                  <a href="https://snapcraft.io/">
-                    <img src="https://assets.ubuntu.com/v1/7f93bb62-snapcraft-logo--web-white-text.svg" alt="Snapcraft.io">
-                  </a>
-              </div>
-              <div id="nav">
-                  <div class="nav-toggle">
-                      <a href="#nav" class="nav-toggle__link open"></a>
-                      <a href="#" class="nav-toggle__link close"></a>
-                  </div>
-                  <ul>
-                      <li><a href="https://snapcraft.io/store/">Store</a></li>
-                      <li><a href="https://snapcraft.io/blog/">Blog</a></li>
-                      <li><a href="https://snapcraft.io/iot">IoT</a></li>
-                      <li><a href="https://snapcraft.io/build">Build</a></li>
-                      <li class='active'><a href="/">Docs</a></li>
-                      <li><a href="https://forum.snapcraft.io/categories" class="external">Forum</a></li>
-                  </ul>
-              </div>
-          </nav>
-        </header>
+          toggle.addEventListener('click', function(e) {
+            toggle.classList.toggle('p-icon--menu');
+            toggle.classList.toggle('p-icon--close');
+            sidebarContent.classList.toggle('u-hide--small');
+          });
+
+          // Add active links to active nav item
+          document.querySelectorAll('.p-sidebar-nav ul a').forEach(function(anchor) {
+            if (anchor.pathname == location.pathname) {
```

Comment:
This isn't working for anchors that have a redirect. I assume this is something that would have to be fixed in discourse?

Reply:
Yes, it would have to be fixed in Discourse.